contracts/Market.sol

```diff
@@ -686,8 +704,7 @@ contract Market is
     bytes32 r,
     bytes32 s
   ) external whenNotPaused {
-    uint256 certificateAmount = this
-      .calculateCertificateAmountFromPurchaseTotal({purchaseTotal: amount});
+    uint256 certificateAmount = amount;
```

Comment:
looks like we could get away with just not assigning this variable at all and passing `amount` directly as the argument in below where `certificateAmount` is passed